docs/UnInstall_IML.md

```diff
@@ -0,0 +1,184 @@
+
+### Optionally stop and remove RabbitMQ
+```
+service rabbitmq-server stop
+yum remove rabbitmq-server
+```
+
+### **Note:**  The IML install creates a repo dir based on a tgz distrubtion and a repo file in /etc/yum.repos.d/ however, both repo dir and repo file are completely removed after installation so they should not need to be deleted now.
+
+
+---
+
+# **Step 2**: Uninstall the IML Agent
+
+### **Remove Host Action** and **Force Remove Action** will do the following on the manager side:
+1. Remove all manager/agent registrations including sessions and queues to prevent any future communication with the host.
+1. Remove the hosts cerificate and revoke it's future use.
+1. Remove database records for the server and related targets.
+
+### The **Remove Host Action** makes changes to the agent server side. 
+
+In the case of **ForceRemoveHostJob**, the agent side is completely untouched. This is the cause of a few problems and should be used when the communication with the agent is no longer possible. If you do use it, this script can be used to clean up the agent side. You will need to get a shell to the agent machine and issue these commands.
+
+### Note also that although the Remove Host Action will remove IML from the agent, it doesn't do a complete job.
+
+### There are some installed artifacts that are not removed. This script can be used to make sure the server is completely uninstalled.
+
+### Some of the actions below are optional.  If you plan on reinstalling the agent, you may choose not to do the optional items.
+
+### Stop and deregister server
+```
+service chroma-agent stop
+/sbin/chkconfig --del chroma-agent
+```
+
+### Remove agent software
+```
+yum remove chroma-agent chroma-agent-management chroma-diagnostics
+```
+
+### Unconfigure NTP
+```
+mv -f /etc/ntp.conf.pre-chroma /etc/ntp.conf
+```
+
+### Erase all cluster information for this server's cluster
+### THIS MEANS THAT OTHER NODES IN THE CLUSTER SHOULD BE REMOVED TOO.
+```
+cibadmin -f -E
+```
+
+### Kill pacemaker and corosync
+```
+killall -9 pacemaker\; killall -9 corosync
+```
+
+### Reset firewall setting
+### Get the multicast port from the corosync setting, and used in the iptables command
+```
+grep 'mcastport' /etc/corosync/corosync.conf
+
+rm -f /etc/corosync/corosync.conf
+
+/sbin/iptables -D INPUT -m state --state new -p udp --dport MCAST-PORT -j ACCEPT
+REMOVE "-A INPUT -m state --state NEW -m udp -p udp --dport MCAST-PORT -j ACCEPT" from /etc/sysconfig/iptables
+REMOVE "--port=MCAST-PORT:udp" from /etc/sysconfig/system-config-firewall
+```
+
+## remove pacemaker and corosync
+```
+yum remove pacemaker-*
+yum remove corosync*
+rm -f /var/lib/heartbeat/crm/* /var/lib/corosync/*
+```
+
+### unconfigure ring1 interface
+```
+ifconfig $SERVER_RING1 0.0.0.0 down
+rm -f /etc/sysconfig/network-scripts/ifcfg-$SERVER_RING1
+```
+
+### unconfigure rsyslog
+```
+Remove lines between # added by chroma-agent\n comments inclusive from /etc/rsyslog.conf See HYD-3090.
```

Comment:
This is no longer relevant and can be removed.  A patch removing the use of `rsyslog` in progress for the RC.

Reply:
removed